main_test.go

```diff
@@ -52,26 +48,15 @@ func TestMain(m *testing.M) {
 	} else {
 		ag.hawkMaxTimestampSkew = time.Minute
 	}
-	// ok this is lame but the contentsignaturepki signer will upload a file dated
-	// at the given second so we don't want anything else to run at the same second
-	// otherwise that file may get rewritten. Easiest way to solve it? Sleep.
-	time.Sleep(time.Second)
-
-	// Initialize a monitor.
-	mo = newMonitor(ag, conf.MonitorInterval)
 
-	// run the tests and exit
-	r := m.Run()
+	t.Cleanup(func() {
+		close(ag.exit)
```

Comment:
Do we also need to remove the rows from the database table here, too? I worry we're just filling up everyone's test db with data (even before this change).

Reply:
That's a good idea, adding.